README.rst

```diff
@@ -4,7 +4,22 @@
 Introduction
 ============
 
-### INTRODUCTION ###
+This package implements some magical improvements compared to the default plone copy/move/rename implementation.
+
+As far as manage to get changes in to Plone/CMFCore this packages is no longer necessary.
+
+Lighting fast move/rename
+--------------------------
+This package provides a improved move/rename implementation.
+The more files you want to move the faster it will be compared to the default Plone implementation.
+
+Example measurement with 1 Folder, which contains 1 Document and 300 Files (almost empty PDFs):
+
+Plone: approx. 80s
+With this package: approx. 8s
+
+PR for this change is open at: XXX (Add URL to PR)
```

Comment:
@maethu can you add a short summary containing _how_ the improvement is implemented?